src/main/java/org/motechproject/prevac/constants/PrevacConstants.java

```diff
@@ -79,7 +79,6 @@
             put("Clinic Location", "location");
             put("Participant Id", "participantId");
             put("Participant Name", "participantName");
-            put("Stage Id", "stageId");
```

Comment:
you have also STAGE constant and it's used in REPORT_COLUMN_WIDTHS and AVAILABLE_LOOKUPS_FOR_VISIT_RESCHEDULE